.prettierrc

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I think we should consider a repo or someplace to look into for the default configuration of prettier as well as necessary configs. Do we already have one?

Reply:
We've discussed it before, but we don't have one yet. Could be a good place to store reusable workflows as well.